webapp/src/ts/modals/edit-user/edit-user-settings.component.html

```diff
@@ -37,7 +37,7 @@
 
     <div class="form-group">
       <label for="language">{{'Language' | translate}}</label>
-      <select id="language" class="form-control" [(ngModel)]="editUserModel.language.code" name="language">
+      <select id="language" class="form-control" [ngModel]="editUserModel.language?.code" (ngModelChange)="editUserModel.language?.code" name="language">
```

Comment:
This was a strange one.  For some reason, Angular did not like it when I used the `?` with the combined `[(ngModel)]`, but it is fine as long as you break it out.  (Got the idea from SO, where this seemed to be just a known limitation of this combined expression...)